payments/models.py

```diff
@@ -121,8 +121,8 @@ class Order(models.Model):
         verbose_name='billed total (USD)',
         help_text='This is the confirmed paid amount from NAU'
     )
-    billed_datetime = models.DateTimeField(
-        null=True,
+    billed_datetime = models.CharField(
+        max_length=300,
         help_text='This is the confirmed date and time of payment',
```

Comment:
Good question. Django convention is no nulls in charfields, use blank string instead.